source/develop/release-management/process/making-an-release.md

```diff
@@ -2,24 +2,168 @@
+
+> Please note that if you want to ship oVirt packages in other distributions you'll need to get permissions there.
+> For example, if you want to ship for Fedora, you need to be package owner on [Fedora Build System](https://koji.fedoraproject.org/koji/).
+>
+> At the time of writing, the oVirt project is focusing only on [CentOS Stream](https://www.centos.org/centos-stream/).
+
+The build process on the [Community Build Service is documented on the CentOS website](https://wiki.centos.org/HowTos/CommunityBuildSystem).
+
+A typycal build is done by taking a src.rpm from the COPR [ovirt/ovirt-master-snapshot Builds](https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-master-snapshot/builds/)
+and rebuilding with:
+
+```bash
+# This builds a scratch (just a test) build for CentOS Stream 9 for oVirt 4.5
+cbs build --scratch virt9s-ovirt-45-el9s <your package .src.rpm>
+
+# This is for CentOS Stream 8:
+cbs build --scratch virt8s-ovirt-45-el8s <your package .src.rpm>
+
+# if the build goes fine you can build for real
+# CentOS Stream 9:
+cbs build virt9s-ovirt-45-el9s <your package .src.rpm>
+
+# CentOS Stream 8:
+cbs build virt8s-ovirt-45-el8s <your package .src.rpm>
+```
+
+Once the builds complete it will be tagged as candidate to be released as `virt8s-ovirt-45-candidate` (for CentOS Stream 8) or `virt9s-ovirt-45-candidate` (for CentOS Stream 9).
+
+In order to publish the build you need to tag it for testing:
+
+```bash
+# CentOS Stream 8
+cbs tag virt8s-ovirt-45-testing <your build without the trailing '.src.rpm'>
+
+#example:
+cbs tag virt8s-ovirt-45-testing ovirt-hosted-engine-ha-2.4.10-1.el8
+```
+
+this will publish the build on https://buildlogs.centos.org/centos/8-stream/virt/ repository.
+
+
+```bash
+# CentOS Stream 9
+cbs tag virt9s-ovirt-45-testing <your build without the trailing '.src.rpm'>
+
+#example:
+cbs tag virt9s-ovirt-45-testing ovirt-hosted-engine-ha-2.4.10-1.el9
+```
+
+this will publish the build on https://buildlogs.centos.org/centos/9-stream/virt/ repository.
+
+
+## Release phase
+
+Once the content in testing has been properly verified and considered good to be released for general availability you'll need to tag your builds for release:
+
+```bash
+# CentOS Stream 8
+cbs tag virt8s-ovirt-45-release <your build without the trailing '.src.rpm'>
+
+#example:
+cbs tag virt8s-ovirt-45-release ovirt-hosted-engine-ha-2.4.10-1.el8
+```
+
+this will publish the build on http://mirror.centos.org/centos/8-stream/virt/ repository and to all CentOS mirrors.
+
+
+```bash
+# CentOS Stream 9
+cbs tag virt9s-ovirt-45-release <your build without the trailing '.src.rpm'>
+
+#example:
+cbs tag virt9s-ovirt-45-release ovirt-hosted-engine-ha-2.4.10-1.el9
+```
+
+this will publish the build on http://mirror.stream.centos.org/SIGs/9-stream/virt/ repository and to all CentOS mirrors.
+
+## Release notes preparation
+
+Release notes can be autogenerated from git logs.
```

Comment:
Will the releng-tools repo still be used? Perhaps clarify how, here or there. If it's only for a release notes, there seem to be some duplication, no? Perhaps we can/should change the script to check cbs/copr directly somehow. Not sure that's easy/possible.

Reply:
I'm adding some notes here and on releng-tools readme. Not sure how to handle the cbs/copr checks in releng-tools scripts.